visma/gui/steps.py

```diff
@@ -32,7 +32,7 @@ def showSteps(workspace):
     Arguments:
         workspace {QtWidgets.QWidget} -- main layout
     """
-    workspace.stepsfigure.suptitle(workspace.output, y=0.98,
+    workspace.stepsfigure.suptitle(workspace.output, x=0.35, y=0.98,
```

Comment:
This is in a way equivalent to having no horizontal scrollbar and x=0.5. Is it not possible to have the scroll position set to center?